src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/ManageBookmarks/ManageBookmarks.xaml.cs

```diff
@@ -0,0 +1,159 @@
+    {
+     
+        public ManageBookmarks()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization. 
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Create new map with a basemap.
+            Map myMap = new Map(Basemap.CreateImageryWithLabels());
+
+            // Set the mapview, map property to the basemap.
+            MyMapView.Map = myMap;
+
+            // Create a set of predefined bookmarks; each one follows the pattern of:
+            // ~ Initialize a viewpoint pointing to a latitude longitude
+            // ~ Create a new bookmark
+            // ~ Give the bookmark a name
+            // ~ Assign the viewpoint
+            // ~ Add the bookmark to bookmark collection of the map
+            // ~ Add the bookmark name to the UI combobox for the user to choose from 
+
+            // Bookmark-1
+            Viewpoint myViewpoint1 = new Viewpoint(27.3805833, 33.6321389, 6000); 
+            Bookmark myBookmark1 = new Bookmark(); 
+            myBookmark1.Name = "Mysterious Desert Pattern"; 
+            myBookmark1.Viewpoint = myViewpoint1; 
+            MyMapView.Map.Bookmarks.Add(myBookmark1); 
+            bookmarkChooser.Items.Add(myBookmark1.Name);
+
+            // Bookmark-2
+            Viewpoint myViewpoint2 = new Viewpoint(37.401573, -116.867808, 6000);
+            Bookmark myBookmark2 = new Bookmark();
+            myBookmark2.Name = "Strange Symbol";
+            myBookmark2.Viewpoint = myViewpoint2;
+            MyMapView.Map.Bookmarks.Add(myBookmark2);
+            bookmarkChooser.Items.Add(myBookmark2.Name);
+
+            // Bookmark-3
+            Viewpoint myViewpoint3 = new Viewpoint(-33.867886, -63.985, 40000);
+            Bookmark myBookmark3 = new Bookmark();
+            myBookmark3.Name = "Guitar-Shaped Forest";
+            myBookmark3.Viewpoint = myViewpoint3;
+            MyMapView.Map.Bookmarks.Add(myBookmark3);
+            bookmarkChooser.Items.Add(myBookmark3.Name);
+
+            // Bookmark-4
+            Viewpoint myViewpoint4 = new Viewpoint(44.525049, -110.83819, 6000);
+            Bookmark myBookmark4 = new Bookmark();
+            myBookmark4.Name = "Grand Prismatic Spring";
+            myBookmark4.Viewpoint = myViewpoint4;
+            MyMapView.Map.Bookmarks.Add(myBookmark4);
+            bookmarkChooser.Items.Add(myBookmark4.Name);
+
+            // Set the initial combobox selection to the lat bookmark added.
+            bookmarkChooser.SelectedIndex = 3;
+
+            // Zoom to the last bookmark.
+            myMap.InitialViewpoint = myMap.Bookmarks.Last().Viewpoint;
+
+            // Hide the controls for adding an additional bookmark.
+            BorderAddBookmark.Visibility = System.Windows.Visibility.Hidden;
+        }
+
+        private void OnBookmarkChooserSelectionChanged(object sender, SelectionChangedEventArgs e)
```

Comment:
This entire function could be replaced with these two lines if Bookmarks are used in the combo box.
````
// Get the selected bookmark and apply the view point to the map
Bookmark selectedBookmark = e.AddedItems[0] as Bookmark;
MyMapView.SetViewpoint(selectedBookmark.Viewpoint);
````